x-pack/platform/plugins/shared/fleet/server/services/package_policy.ts

```diff
@@ -739,59 +758,82 @@ class PackagePolicyClientImpl implements PackagePolicyClient {
 
   public async get(
     soClient: SavedObjectsClientContract,
-    id: string
+    id: string,
+    options: PackagePolicyClientGetOptions = {}
   ): Promise<PackagePolicy | null> {
-    const savedObjectType = await getPackagePolicySavedObjectType();
-    const packagePolicySO = await soClient.get<PackagePolicySOAttributes>(savedObjectType, id);
-    if (!packagePolicySO) {
```

Comment:
I think that code was probably not correct as .get seems to throw on not found, maybe something we should keep

Reply:
Oh... So `return null` is not correct? if not found, it should throw? if thats the case, then I'll just update the new implementation below (line 776) to set `ignoreMissing` to `false` when calling `getByIDs()`

Reply:
Yes it seems it currently throw and the return null part is dead code.

Reply:
👍 . I'll set `ignoreMissing` to `true` and that should give us the same behavior